index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Randall Mowen | INFO 2439 | Mobile Home Page</title>
+    <meta charset="utf-8">
+    <link rel="stylesheet" type="text/css" href="_css/Mowen_a1.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css" />
+    <script src="http://code.jquery.com/jquery-1.11.1.min.js"></script>
+    <script src="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
```

Comment:
consider linking first to jquery css then to your custom css so that jquery doesn't overwrite your custom styles.